components/spreadsheet/functions-formulas.md

```diff
@@ -0,0 +1,304 @@
+---
+title: Functions and Formulas
+page_title: Spreadsheet - Functions and Formulas
+description: Learn about all built-in functions supported by the Telerik UI for Blazor Spreadsheet.
+slug: spreadsheet-functions-formulas
+tags: telerik,blazor,spreadsheet
+published: True
+position: 30
+---
+
+# Spreadsheet Functions and Formulas
+
+This article lists the supported built-in functions and formulas by Telerik Spreadsheet for Blazor.
+
+
+## Formulas
+
+A Spreadsheet formula starts with the equal sign `=` and can include:
+
+* References to cells and ranges
+* Constants and literals
+* Operators
+* [Functions](#function-list)
+
+For more information, you can refer to:
+
+* [Microsoft Formula documentation](https://support.microsoft.com/en-us/office/overview-of-formulas-in-excel-ecfdc708-9162-49e8-b993-c311f47ca173)
+* [Microsoft Operator documentation](https://support.microsoft.com/en-gb/office/calculation-operators-and-precedence-in-excel-48be406d-4975-4d31-b2b8-7af9e0e2878a)
+
+To use `=` as a string at the beginning of a cell, start with an apostrophe: `'=`.
+
+
+## Function List
```

Comment:
```suggestion
## Functions List
```

Reply:
Nouns should be singular when used as an adjective.